website/src/data/navigation.yaml

```
@@ -62,6 +62,8 @@
       path: /deployment-examples/surge
     - title: Deploying to Cloudflare Pages
       path: /deployment-examples/cloudflare
+    - title: Deploying to Google Cloud with Cloud Storage
+      path: /deployment-examples/google-cloud-storage
```

To keep it consistent with the similar Azure example

```suggestion
    - title: Deploying to Google Cloud with Static Web Apps
      path: /deployment-examples/google-cloud-static-web-apps
```

You also need to rename the file.